packages/next/taskfile.js

```diff
@@ -25,7 +25,7 @@ export async function nextbuild (task, opts) {
 }
 
 export async function client (task, opts) {
-  await task.source(opts.src || 'client/**/*.+(js|ts|tsx)').typescript().target('dist/client')
+  await task.source(opts.src || 'client/**/*.+(js|ts|tsx)').typescript({module: 'commonjs'}).target('dist/client')
```

Comment:
We'll have to do `target: 'es5'` for this one too as otherwise we'll ship newer syntax to older browsers.